.github/workflows/tests_linux.yml

```diff
@@ -207,6 +208,47 @@ jobs:
               -DENABLE_BLAS=ON \
               -DPL_BACKEND=${{ matrix.pl_backend }} \
               -DBUILD_TESTS=ON \
+              -DENABLE_LAPACK=ON \
+              -DCMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION) \
+              -DENABLE_COVERAGE=ON
+            cmake --build ./Build
+            cd ./Build
+            mkdir -p ./tests/results
+            for file in *runner ; do ./$file --order lex --reporter junit --out ./tests/results/report_$file.xml; done;
+            lcov --directory . -b ../pennylane_lightning/core/src --capture --output-file coverage.info
+            lcov --remove coverage.info '/usr/*' --output-file coverage.info
+            mv coverage.info coverage-${{ github.job }}-${{ matrix.pl_backend }}.info
```

Comment:
why are these coverage commands added here?

Reply:
Actually, it's kind of GA issue. This difference should be shown in the `cpptestswithOpenBLASNoLAPACK` section.